### PR TITLE
[SG-632] - Change forwarded providers radio buttons list to dropdown

### PR DESCRIPTION
**Pull request by LRNcardozoWDF**
## Type of change

```
- [ ] Bug fix
- [X] New feature development
- [ ] Tech debt (refactoring, code cleanup, dependency upgrades, etc)
- [ ] Build/deploy pipeline (DevOps)
- [ ] Other
```

## Objective

Change the forwarded provider services radio buttons list to a dropdown list alphabetically ordered.

## Code changes

- **apps/browser/src/popup/generator/generator.component.html**
   **apps/desktop/src/app/vault/generator.component.html**
   **apps/web/src/app/tools/generator.component.html:** Changed radio button list to dropdown. Added attribute roles in order to support accessibility.
- **libs/angular/src/components/generator.component.ts:** Organised the initialization of the array in a private method. Besides ordering the array also adds an empty option when there's no data.

## Screenshots

**Before**
image

**Now**
image
image

## Before you submit

- Please add **unit tests** where it makes sense to do so (encouraged but not required)
- If this change requires a **documentation update** - notify the documentation team
- If this change has particular **deployment requirements** - notify the DevOps team

### CODE REVIEW

File: apps/web/src/app/tools/generator.component.html [hunk 1]
```diff
@@ -266,23 +266,18 @@ <h1>{{ "generator" | i18n }}</h1>
...
-          (change)="saveUsernameOptions()"
-          [checked]="usernameOptions.forwardedService === o.value"
-        />
-        <label class="form-check-label" for="forwardtype_{{ o.value }}">
+      <select
+        id="ForwardTypeDropdown"
+        name="ForwardType"
+        [(ngModel)]="usernameOptions.forwardedService"
+        (change)="saveUsernameOptions()"
+      >
```
Comment (JaredSnider-Bitwarden) [inline apps/web/src/app/tools/generator.component.html:276]: @LRNcardozoWDF, the web select needs additional styling to match the standard styling. 

Currently:
image

Standard (from the Generator > Import Data tool):
image

You could solve by either using the `form-control and w-auto` class or putting it in a div w/ `col-4 `class similar to the API key input below the select.

Comment (differsthecat) [reply apps/web/src/app/tools/generator.component.html:276]: I'd avoid using bootstrap classes like `col-4` if at all possible as we are migrating to [tailwind](https://contributing-internal.bitwarden.com/external/code-style/tailwind/) in web!

Comment (JaredSnider-Bitwarden) [reply apps/web/src/app/tools/generator.component.html:276]: Browser & Desktop looked and worked great!

Comment (LRNcardozoWDF) [reply apps/web/src/app/tools/generator.component.html:277]: Thank you for catching this, fixed as you suggested.

**Review by JaredSnider-Bitwarden — APPROVED** [review]

**Pull request merged**